### PR TITLE
Updating Current Face Payment Copywriting to Match with Wireframe

### PR DESCRIPTION
**Pull request by sonadztux**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

Copywriting untuk Face Payment belum sesuai dengan wireframe terbaru dari Tim Produk

## What is the new behavior?

Menyesuaikan copywriting untuk Face Payment sesuai dengan wireframe terbaru dari Tim Produk

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: frontend/app/components/modules/FaceEnrollment/FaceEnrollment.tsx [hunk 1]
```diff
@@ -129,8 +129,13 @@ export const FaceEnrollment = ({ openModal, payload, nextStep }: Props) => {
       ) : (
         <>
           <div className={styles.subtitle}>
-            <h2>Say Cheese!</h2>
-            <p>Take a selfie to register your payment account</p>
+            <h2>Take a selfie!</h2>
+            <p>
+              Ensure your face position is in the oval area <br />
+            {/* </p>
```
Comment (gregoriusjimmy) [inline frontend/app/components/modules/FaceEnrollment/FaceEnrollment.tsx:135]: ini nan

File: frontend/app/components/modules/PaymentForm/PaymentForm.tsx [hunk 2]
```diff
@@ -104,6 +101,7 @@ export const PaymentForm = ({ onNextStep, onInvalidSession }: Props) => {
             }}
             errors={errors}
           />
+          <span></span>
```
Comment (gregoriusjimmy) [inline frontend/app/components/modules/PaymentForm/PaymentForm.tsx:104]: ini

File: frontend/app/components/modules/FaceEnrollment/FaceEnrollment.tsx (hunk 1 above)
Comment (rizkybaihaqy) [inline frontend/app/components/modules/FaceEnrollment/FaceEnrollment.tsx:135]: apus aja ini kalo ga dipake

File: frontend/app/components/modules/PaymentForm/PaymentForm.tsx (hunk 2 above)
Comment (rizkybaihaqy) [inline frontend/app/components/modules/PaymentForm/PaymentForm.tsx:104]: ini span kosong untuk apa nan?

File: frontend/app/components/modules/FaceEnrollment/FaceEnrollment.tsx (hunk 1 above)
Comment (sonadztux) [reply frontend/app/components/modules/FaceEnrollment/FaceEnrollment.tsx]: done https://github.com/nodefluxio/weber/pull/186/commits/a214fc34a44126dbafafb1448bccb177d5da0ee5

File: frontend/app/components/modules/PaymentSetup/PaymentSetup.tsx [hunk 3]
```diff
@@ -42,11 +42,10 @@ export const PaymentSetup = ({ onChange }: Props) => {
 
   return (
     <div className={styles.modalPayment}>
-      <h2>Set up your minimum payment</h2>
+      <h2>Set up your payment limit</h2>
       <span className={styles.value}>{value.toLocaleString()}</span>
       <p>
-        If the payment exceeds the minimum payment, you are required to input
-        pin
+        You're required to enter a PIN if the total purchase of items exceeds your entered amount
```
Comment (gregoriusjimmy) [inline frontend/app/components/modules/PaymentSetup/PaymentSetup.tsx:48]: gabisa pake tanda petik

File: frontend/app/components/modules/PaymentForm/PaymentForm.tsx (hunk 2 above)
Comment (sonadztux) [reply frontend/app/components/modules/PaymentForm/PaymentForm.tsx]: done https://github.com/nodefluxio/weber/pull/186/commits/95d97f23ddfdb3178cad11fb6332b65c83a4275a

**Comment by lazuardyk** [comment]
Screenshot_2044

File: frontend/app/components/modules/PaymentForm/PaymentForm.tsx [hunk 4]
```diff
@@ -71,14 +71,11 @@ export const PaymentForm = ({ onNextStep, onInvalidSession }: Props) => {
...
-        <h2>Register Yourself</h2>
+        <h2>Dummy Wallet Registration</h2>
         <p>
-          Before making any transaction using face payment, you must register
-          your e-wallet account first
-        </p>
-        <p>
-          *Please remember your phone number, since it will be used when
-          you did the transaction
+          It's only dummy wallet registration, no need a card or top up anything from you <br />
```
Comment (gregoriusjimmy) [inline frontend/app/components/modules/PaymentForm/PaymentForm.tsx:76]: gabisa tanda petik 1 , solusi  you&apos;re free

Comment (sonadztux) [reply frontend/app/components/modules/PaymentForm/PaymentForm.tsx]: done https://github.com/nodefluxio/weber/pull/186/commits/c9ddf7dbbbce091f316cedad0719e60be2b8f3f8

File: frontend/app/components/modules/PaymentSetup/PaymentSetup.tsx (hunk 3 above)
Comment (sonadztux) [reply frontend/app/components/modules/PaymentSetup/PaymentSetup.tsx]: done https://github.com/nodefluxio/weber/pull/186/commits/c9ddf7dbbbce091f316cedad0719e60be2b8f3f8

**Review by gregoriusjimmy — APPROVED** [review]

**Review by lazuardyk — APPROVED** [review]

**Pull request merged**